Answers.md

```diff
@@ -1,2 +1,35 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+# 1. Describe some of the differences between `.forEach` & `.map`. 
+     
+     .forEach is a loop that applies an operation to each list item. ForEach does not return something.
```

Comment:
I love this. I would add onto it, that it's primary just a function to iterate over a list and callback the elements with the index. Just maybe swapping the term 'operation' to 'callback' seems a bit better.